src/components/App.tsx

```diff
@@ -923,6 +924,14 @@ function App() {
 
   useDebouncedWindowResize(autoZoomIfNecessary, 100);
 
+  /*//////////////////////////////////////////////////////////////
+                        CHAT RESIZE LOGIC
+  //////////////////////////////////////////////////////////////*/
+
+  const [savedChatSize, setSavedChatSize] = useLocalStorage<string>(
+    SAVED_CHAT_SIZE_LOCAL_STORAGE_KEY
+  );
```

Comment:
wait will this be a string? is ref.style.width a value like `533px` or `533`

Reply:
It's a string representing a percentage value, so this is correct

<img width="101" alt="Screenshot 2023-04-04 at 4 15 20 PM" src="https://user-images.githubusercontent.com/25194960/229924238-127fd775-57ee-4ad6-8c4c-3ef9765ecaad.png">
<img width="417" alt="Screenshot 2023-04-04 at 4 14 35 PM" src="https://user-images.githubusercontent.com/25194960/229924071-89e9d924-3155-405b-a7a9-a013642e2c88.png">

Reply:
ahh interesting — sg